client/extensions/woocommerce/app/settings/payments/payments-off-site.js

```diff
@@ -25,7 +52,17 @@ class SettingsPaymentsOffSite extends Component {
 							'information'
 						)
 					} />
-				<Card></Card>
+					<ListTable>
+						<ListHeader>
+							<ListTd width="30">Method</ListTd>
```

Comment:
You named the components `ListTd`, `ListTable` etc. I think they should have more semantic names, instead of blatantly giving away that they represent an HTML `<table>` underneath. What about `ListRowField` instead of `ListTd`, for example?